Docker/README.md

```diff
@@ -0,0 +1,36 @@
+# docker-logitechmediaserver
+
+Docker image for Logitech Media Server
+
+Run with:
+
+```
+docker run -t -i --rm=true --net="bridge" \
+      -v "<somewhere>":"/config":rw \
+      -v "<somewhere>":"/music":ro \
+      -v "<somewhere>":"/playlist":ro \
+      -v "/var/run/dbus":"/var/run/dbus":rw \
```

Comment:
We can't expect dbus to be available on any host, can we? Eg. running Docker on Windows this wouldn't work?

Reply:
In the end is a user decision what he puts in the run command. We can remove this from readme sure.

Reply:
What is dbus for? (Please excuse my ignorance...)

Reply:
For the airplay plugin to work.